test/routing-tables.test.js

```diff
@@ -547,114 +562,6 @@ describe('RoutingTables', function () {
       }, /TypeError: RoutingTables#toJSON maxPoints must be a positive number/)
     })
   })
-
-  describe('findBestHopForDestinationAmount', function () {
-    it('finds the best route when there is one path', function () {
```

Comment:
Yes, see https://github.com/interledgerjs/ilp-routing/pull/46/files/fbd64b637b1ee8a8085419782d7293c3acb1ca47#diff-f41631b9482b1a98863e2c9f66b37558L240 for the tests